languages/csharp-dotnet-2.2/template_binary/__ALGO__.csproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
```

Comment:
Obviously template_binary isn't necessary as I was using it just for testing. It could be an alternative template though - the easy decision is to just remove it, thoughts? @kennydaniel

Reply:
Removed.